package.json

```diff
@@ -24,21 +28,45 @@
     "container",
     "Google Container Engine API"
   ],
+  "contributors": [
+    "Alexander Fenster <fenster@google.com>",
+    "Ethan Bao <cbao@google.com>",
+    "Google API Publisher <googleapis-publisher@google.com>"
+  ],
+  "scripts": {
+    "publish-module": "node ../../scripts/publish.js container",
+    "smoke-test": "mocha smoke-test/*.js --timeout 5000",
```

Comment:
We have `smoke-test` and `system-test` here. I'm pretty sure @lukesneeringer mentioned before that smoke-test was going to be removed in favor of system-test to match our other APIs.

Either way, though, the `system-test` command is not in use here, so could probably be removed.

Reply:
I talked to Luke about that. Smoke-test is some very basic system test that is being generated automatically.  We basically have two options here: we either move smoke test to system test and pretend that it's a real system test (and run it using "npm run system-test"), or leave it as a smoke test (and system test might appear later if someone takes time and implements them). I personally prefer leaving smoke test as generated to make the process of setting up new packages as easy as possible, with little or no manual files moving.

As for having "system-test" script in package.json that does not work (for now), I don't see any problem here. It will save us some time in future when system tests are finally added (the same for samples).

Does that make sense for you?

Reply:
Thanks for the explanation. What if we had the system-test command run the smoke tests as well? The win would be one command to test against the actual API. Alternatively, the system-test directory could hold a file called "generated.js" or something similar that will be picked up by the system-test command already. Then hand-written tests could be placed in system-test/api.js.

It would also be nice if the smoke tests used the same form of auth as the system tests (ADC), and didn't require learning and setting a new env var.